### ISSUE TITLE
Model parity

### ISSUE THREAD

**Issue by eladBenedict** [tensorflow/ranking#181]
Hi,

I have a model that scores a single item (each row has both context and target features) and is currently trained with cross-entropy loss.
I'm using this model as the scoring function in the TF-Ranking framework (also, with cross-entropy loss), and as a sanity check, expected to get the same results (e.g. loss over the test set) from both models.

The hyper-parameters and configuration are the same, but the resultant loss is significantly different (0.48 for non TF-Ranking model, and 0.62 for the TF-Ranking model).

1. Is it correct to expect both models to perform the same (assuming there's plenty of data) or is there something inherently different between them?
2. Any idea what may be the cause for such a performance gap?

Thanks,
Elad

**Comment by xuanhuiwang** [comment]
How do you measure the loss? Do you use sigmoid_cross_entropy_loss in tf-ranking?

The only difference that I see is the batching of the data points are different, assuming you are using ELWC in tf-ranking, but tf.example in non-tf-ranking model.

**Comment by eladBenedict** [comment]
Thanks @xuanhuiwang - managed to achieve parity.
Yes, I used sigmoid cross entropy, and ensured that batching is done identically
